pkg/ebpf/probes/probes.go

```diff
@@ -266,6 +274,73 @@ func (p *traceProbe) autoload(module *bpf.Module, autoload bool) error {
 	return enableDisableAutoload(module, p.programName, autoload)
 }
 
+//
+// uProbe
+//
+
+type uProbe struct {
```

Comment:
Idea for the future, I think we can have this struct have a `Trigger` method of it's own, then instead of manually building the `symbolName` we can also generate it (of course matching to an existing bpf uprobe code).

This could (possibly) also help with a more concrete method of event invocation, an invoked event from a trigger would have its context stored "by definition" through this `Trigger`.

Reply:
I would wait for more uprobe usage needs (specially those unrelated to tracee binary itself, like these ones) and then define something else if needed.